packages/browser/src/ClientAuthentication.ts

```diff
@@ -123,6 +126,47 @@ export default class ClientAuthentication {
     return this.sessionInfoManager.getAll();
   };
 
+  getCurrentIssuer = async (): Promise<string | null> => {
+    const currentSessionId = window.localStorage.getItem(KEY_CURRENT_SESSION);
+    if (currentSessionId === null) {
+      return null;
+    }
+    const sessionInfo = await this.sessionInfoManager.get(currentSessionId);
+    // Several session information are required in order to validate that the ID
+    // token in storage hasn't been tampered with, and has actually been issued
+    // by the issuer present in storage.
```

Comment:
"Information" is uncountable in English :)

```suggestion
    // Several types of session data are required in order to validate that the ID
    // token in storage hasn't been tampered with, and has actually been issued
    // by the issuer present in storage.
```

Reply:
Ah right, I always forget ^^